{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Base Image
+FROM python:3.6.6-alpine3.7
```

Comment:
@gianlucabiccari a few notes here:

1. It's best to not overspecify the version number: 3.6 is enough detail.
2. I like alpine, but in the Python ecosystem your dependencies will install _much_ faster if you use a Debian base (because there are prebuilt wheels for Debian). I use `python:3.6-slim-stretch`, which is comparable in size to alpine and gives us that speed advantage.

Reply:
I am ok moving to `python:3.6-slim-stretch`, in fact I think we should uniform that across teams that build python project.